app.py

```diff
@@ -125,22 +116,33 @@ def opt_out_of_experiments():
 @app.route(f"{URL_PREFIX}/unsubscribe")
 @auth.requires_login
 def unsubscribe():
-    account_id = auth.get_account_id()
-    with DB_ENGINE.connect() as conn:
-        account_repo = DbAccountRepository(conn)
-        account_repo.remove_subscription_for_account(account_id)
-        conn.commit()
-
-    return redirect(url_for("home", error_description="Sorry to see you go. You have been unsubscribed"))
+    return render_template("pre_unsubscribe.html")
 
 
-@app.route(f"{URL_PREFIX}/email_unsubscribe/<newsletter_id>")
-def email_unsubscribe(newsletter_id):
+@app.route(f"{URL_PREFIX}/pre_unsubscribe", methods=["POST"])
+@auth.requires_login
+def pre_unsubscribe():
+    main_menu_option = request.form.get("main-menu")
+    unsubscribe_menu_option = request.form.get("unsubscribe-menu")
+    error_description = "Please reach out to poprox admins to complete this request."
```

Comment:
I _think_ this is out-of-date after our last discussion where we decided it would make more sense to soft-delete in the database rather than tell them to email us?